src/contrib/building-sssd.rst

```diff
@@ -1,90 +1,90 @@
 Building SSSD
-=============
+#############
 
-Starting with SSSD 1.10 beta, we now include a set of helper aliases and
-environment variables in SSSD to simplify building development versions of
-SSSD on Fedora. To take advantage of them, use the following command:
+SSSD is using `autoconf`_ and `automake`_ build systems. If you are familiar
+with these tools, you already know what to do. If you are not, these are the
+commands you need to execute.
 
-.. code-block:: bash
+.. _autoconf: https://www.gnu.org/software/autoconf
+.. _automake: https://www.gnu.org/software/automake
 
-    . /path/to/sssd-source/contrib/fedora/bashrc_sssd
+.. code-block:: console
 
-To build SSSD from source, follow these steps:
+    [sssd]$ autoreconf -if
+    [sssd]$ ./configure --enable-nsslibdir=/lib64 --enable-pammoddir=/lib64/security --enable-silent-rules
+    [sssd]$ make
 
-- Install necessary tools:
+The ``autoreconf`` command will create a ``configure`` script. This script
+allows you to modify the resulting build with several flags, it also makes sure
+that all required dependencies are available. SSSD requires lots of build
+dependencies that helps us integrate with the identity and authentication
+systems. Chances are that you don't have all the dependencies installed. In such
```

Comment:
s/Chances are that you don't have all the dependencies installed/Your system may not have all necessary dependencies installed/